py3status/py3.py

```diff
@@ -93,3 +98,31 @@ def replace_fn(match):
 
         format_string = re.sub('\{[^}]*\}', replace_fn, format_string)
         return format_string.format(**param_dict)
+
+    def check_commands(self, cmd_list):
+        """
+        Checks to see if commands in list are available using `which`.
+        Returns the first available command.
+        """
+        devnull = open(os.devnull, 'w')
+        for cmd in cmd_list:
+            c = shlex.split('which {}'.format(cmd))
+            if call(c, stdout=devnull, stderr=devnull) == 0:
+                return cmd
+
+    def play_sound(self, sound_file):
+        """
+        Plays sound_file if possible.
+        """
+        self.stop_sound()
+        cmd = self.check_commands(['paplay', 'play'])
+        if cmd:
+            c = shlex.split('{} {}'.format(cmd, sound_file))
+            self._audio = Popen(c, shell=True)
+
+    def stop_sound(self):
+        """
+        Stops any currently playing sounds for this module.
+        """
+        if self._audio:
+            self._audio.kill()
```

Comment:
It is initialized as `None` otherwise it is the result of `Popen()` which does have `kill()`  I should probably set it to `None` following the `kill()` call.